aws/asg-ebs/README.md

```diff
@@ -0,0 +1,71 @@
+# aws/asg-ebs - Auto Scaling Group with EBS mount
+This module is used to create an auto scaling group launch configuration and
+an auto scaling group that uses the configuration.  An EBS file system is mounted.
+During transition from EFS to EBS, an EFS file system is also mounted.
+
+## What this does
+
+ - Create launch configuration named after `app_name` and `app_env`
+ - Create auto scaling group of defined size and distribute instances across `aws_zones`
+
+## Required Inputs
+
+ - `app_name` - Name of application, ex: Doorman, IdP, etc.
+ - `app_env` - Name of environment, ex: prod, test, etc.
+ - `ami_id` - ID for AMI to be used.
+ - `aws_instance` - A map containing keys for `instance_type`, `volume_size`, `instance_count`
+ - `aws_region` - Default AWS region
+ - `aws_access_key` - Access key to allow access via the AWS CLI
+ - `aws_secret_key` - Secret access key to allow access via the AWS CLI
+ - `private_subnet_ids` - A list of private subnet ids to identify VPC subnets for placement
+ - `default_sg_id` - VPC default security group ID to add instances to
+ - `ecs_instance_profile_id` - IAM profile ID for ecsInstanceProfile
+ - `ecs_cluster_name` - ECS cluster name for registering instances
+ - `efs_dns_name` - DNS name of the EFS file system to be mounted
+ - `mount_point` - Full path to directory on which to mount the EFS file system
+
+## Transitional Inputs
+
+ - `ebs_device` - Device name for EBS volume, e.g., "/dev/sdh"
+ - `ebs_mount_point` - Full path to directory on which to mount the file system contained in the EBS volume
+ - `ebs_vol_id` - EBS volume ID
+ - `ebs_mkfs_label` - Label for filesystem. Default: `Data`
+ - `ebs_mkfs_labelflag` - Flag preceding the label name in the mkfs command. Default: `-L`
+ - `ebs_mkfs_extraopts` - Extra options to pass to the mkfs command. Default: ""
+ - `ebs_fs_type` - Type of filesystem to create. Default: `ext4`
+
```

Comment:
`ebs_mountopts` isn't listed here, should it be?

Reply:
Yes, thanks.